src/core/zowe/core_for_zowe_sdk/constants.py

```diff
@@ -14,6 +14,7 @@
     "SecureValuePrefix": "managed by ",
     "TsoSessionNotFound": "IZUG1126E",
     "ZoweCredentialKey": "Zowe-Plugin",
+    "MaxAllocationQuantity": 16777215,
```

Comment:
Thanks for moving this constant into the z/OS Files SDK 👍 

Please remove this duplicate version of the constant from the Core SDK.